app/views/restaurants/index.html.haml

```diff
@@ -18,3 +12,4 @@
       %p
         = restaurant.address
 #map{style: 'width: 800px; height: 400px', data: {test: {env: "#{Rails.env.test?}"}}}
+#restaurants_addresses{data: {addresses: "#{@restaurants.to_json}"}}
```

Comment:
You can remove this - we don't need it anymore and deleted it last time we merged :)

Reply:
@Sajaas it's removed now